interfaces/seccomp/template.go

```diff
@@ -185,8 +185,9 @@ inotify_init
 inotify_init1
 inotify_rm_watch
 
-# Needed by shell
-ioctl
+# TIOCSTI allows for faking input (man tty_ioctl)
+# TODO: this should be scaled back even more
```

Comment:
@zyga - I was talking about ioctl in general instead of TIOCSTI. This closes the TIOCSTI issue which is what the PR is for. For ioctl in general, it would require looking at everywhere ioctl is used and all ioctls and see if there is anything else to block vs allowing only a subset of common ioctls.